data/github/issue_context.yaml

```diff
@@ -1,5 +1,16 @@
 "integration: demo": >-
   This is a demo integration.
+"integration: unifiprotect": >-
+  Please make sure this isn't a duplicate issue by searching the existing issues, especially keep an eye on the latest issues and pull requests.
+
+  https://github.com/home-assistant/core/issues?q=%20label%3A%22integration%3A%20unifiprotect%22%20
+
+  Then make sure your report includes the following information:
+
+  1. UniFi OS version
+  2. Protect version
+  3. Hardware running Protect (Cloud Key Gen2+, UDM-Pro, etc.)
+  4. If applicable, error logs
```

Comment:
Let's make this a link to https://www.home-assistant.io/docs/configuration/troubleshooting/#debug-logs-and-diagnostics